src/RageTextureManager.cpp

```diff
@@ -254,27 +254,8 @@ void RageTextureManager::DeleteTexture( RageTexture *t )
 		m_texture_ids_by_pointer.erase(id_entry);
 		return;
 	}
-	else
```

Comment:
Hmm I don't think we need these changes.

It seems like it first checks `m_texture_ids_by_pointer` for the existence of `t`, else if it doesn't find it then it checks the `m_mapPathToTexture` for `t` which are different?

Reply:
This is another case of `FAIL_M()` exiting immediately. All the code after it will theoretically not be executed, which is why the compiler flagged it, and I removed it.

Reply:
Ah I seeee. Gotcha seems good then.